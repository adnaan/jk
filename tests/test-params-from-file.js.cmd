jk run --parameters %b.json %f
