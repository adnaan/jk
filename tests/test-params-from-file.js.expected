{
  "myBoolean": true,
  "myNumber": 2,
  "myObject": {
    "b": true,
    "message": "success",
    "o": {
      "foo": "bar",
      "xxx": "yyy"
    },
    "s": "bar"
  },
  "myString": "success"
}
